recipe/doc/end_user_run_guide.md

```diff
@@ -0,0 +1,121 @@
+# Guide for End-Users Running CUDA Code
+
+This guide is for people who wish to use conda environments to run CUDA code.
+
+## Prerequisites
+
+To run CUDA code, you must have an NVIDIA GPU on your machine and you must install the CUDA drivers.
+Note that CUDA drivers _cannot_ be installed with conda and must be installed on your system using an appropriate installation method.
+See the [CUDA documentation](https://docs.nvidia.com/cuda/) for instructions on how to install ([Linux](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html), [Windows](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html)). 
+
+## Installing CUDA
```

Comment:
Installing CUDA packages in a CPU-only environment is also allowed and we should mention it.
```suggestion

Note that the CUDA packages are designed to be installable in a *CPU-only* environment (i.e. no CUDA driver or GPU installed). In this case, you need use the `cuda-version` package as discussed below to control the packages to install, and keep in mind that any CUDA code will not be executable in such environments. The main purpose of this support is to allow smoking gun tests in a resource-limited (e.g. public CI) environment.

## Installing CUDA
```

Reply:
I see it's already mentioned in the FAQ, modified my suggestion above.